### ISSUE TITLE
zsh: command not found: wallpapper

### ISSUE THREAD

**Issue by fuckinalright** [mczachurski/wallpapper#41]
It always show zsh: command not found: wallpapper

**Comment by mczachurski** [comment]
Could you provide more information? Which command do you use and how the logs from these commands looks like?

**Comment by farseer00** [comment]
I might be able to provide some insight on this. I am having the same issue. However, I'm on an ARM Mac, so my experience might be a little different. I used the manual install method, since homebrew isn't supported on ARM Macs yet. I'm using macOS Big Sur on an ARM MacBook Air.

After installing Xcode, I was able to run the first three commands without any issues:
`git clone https://github.com/mczachurski/wallpapper.git`
`cd wallpapper`
`swift build --configuration release`

It's the last command that I'm having trouble with. I had to change the command to the following:
`sudo cp .build/arm64-apple-macosx/release/wallpapper /usr/local/bin`

When run, the command shows no errors in Terminal, but shows some interesting output in the system.log (see attached file)
[system log.txt](https://github.com/mczachurski/wallpapper/files/5684171/system.log.txt)

I discovered that I can use a workaround by manually locating the wallpapper executable file and executing the following:
`cd Dynamic_Wallpaper_Folder`
`/Users/mymac/wallpapper/.build/arm64-apple-macosx/release/wallpapper -i wallpapper.json`

Using this method generates the output.heic file without any problems.

Hope this helps.
